packages/dtd-catalog-exporter/src/models/converters.ts

```diff
@@ -30,14 +31,25 @@ export function toPublicEService(
     throw genericError(`Producer for e-service ${eservice.id} not found`);
   }
 
+  const { producerFiscalCode, producerIpaCode } = match(producer.externalId)
+    .with({ origin: "IPA" }, ({ value }) => ({
```

Comment:
You can reuse the const defined in `models`  
```suggestion
    .with({ origin: PUBLIC_ADMINISTRATIONS_IDENTIFIER }, ({ value }) => ({
```